test/integration/targets/synchronize_docker/aliases

```diff
@@ -0,0 +1 @@
+non_local
```

Comment:
Can the CI run a test depending on a docker transport at all? [connection_docker](https://github.com/ansible/ansible/blob/devel/test/integration/targets/connection_docker/aliases) isn't tagged for running on the CI either and it has the same requirements.